utils/scripts/ocm/ocm.py

```diff
@@ -843,14 +844,56 @@ def update_notification_email_address(self, addon_name, email_address, exit_on_f
         ret = execute_command(cmd)
         if ret is None:
             log.info("Failed to update email address to {} addon on cluster "
-                  "{}".format(addon_name, self.cluster_name))
+                     "{}".format(addon_name, self.cluster_name))
             if exit_on_failure:
                 sys.exit(1)
             else:
                 return False
         else:
             return ret
 
+    def install_openshift_isv(self, operator_name, channel, source, exit_on_failure=True):
+        replace_vars = {
+            "ISV_NAME": operator_name,
+            "CHANNEL": channel,
+            "SOURCE": source
+        }
+        template_file = "install_isv.jinja"
+        output_file = "install_isv.yaml"
+        self._render_template(template_file, output_file, replace_vars)
+        cmd = ("oc apply -f {} ".format(output_file))
+        ret = execute_command(cmd)
+        if ret is None:
+            log.info("Failed to apply install isv subscription")
+            if exit_on_failure:
+                sys.exit(1)
+            else:
+                return False
+        else:
+            return ret
+
+    def wait_for_isv_installation_to_complete(self, isv_name, namespace='openshift-operators', timeout=300):
+        count = 0
+        check_flag = False
+        cmd = ("oc get csv -n openshift-operators $(oc get csv -n {} | grep -i {} | awk '{}') -o json "
+               "| jq '.status.phase'".format(namespace, isv_name, '{print $1}'))
```

Comment:
it might be worth to create a kw/function to get the csv status (for future)

Reply:
Sure ..we could do that